luxonis_train/nodes/heads/ddrnet_segmentation_head.py

```diff
@@ -0,0 +1,106 @@
+import logging
+
+import torch.nn as nn
+from torch import Tensor
+
+from luxonis_train.nodes.base_node import BaseNode
+from luxonis_train.nodes.blocks import ConvModule
+from luxonis_train.utils.types import LabelType
+
+logger = logging.getLogger(__name__)
+
+
+class DDRNetSegmentationHead(BaseNode[Tensor, Tensor]):
+    in_height: int
+    n_classes: int
+    tasks: list[LabelType] = [LabelType.SEGMENTATION]
+    attach_index: int = 0
```

Comment:
It's safer for the `attach_index` to be negative for better compatibility with other backbones. The normal head would be attached to `-1` and the auxiliary to `-2`.